scripts423.sql

```diff
@@ -0,0 +1,7 @@
+SELECT student.name, student.age, faculty.name
+FROM student
+         INNER JOIN faculty ON student.faculty_id = faculty.id
```

Comment:
```suggestion
         LEFT JOIN faculty ON student.faculty_id = faculty.id
```
потому что надо вывести информацию обо ВСЕХ студентах, а не ТОЛЬКО ТЕХ, у которых есть факультет